test/Microsoft.Dnx.Runtime.Tests/VersionUtilityFacts.cs

```diff
@@ -129,6 +129,7 @@ public void GetNearestPicksMostCompatibleItem(string input, string frameworks, s
         }
 
         [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
For that very reason I would say remove it - so that you are aware when suddenly for some reason tests get run on Mono on Windows...